src/main/java/org/folio/entlinks/controller/converter/AuthoritySourceFileMapper.java

```diff
@@ -145,7 +145,11 @@ private static String getHostPath(URL url) {
   }
 
   private static void setUrlProperties(String baseUrlDto, AuthoritySourceFile target) {
-    if (StringUtils.isBlank(baseUrlDto)) {
+    if (baseUrlDto == null) {
+      return;
+    }
```

Comment:
```suggestion
    if (StringUtils.isBlank(baseUrlDto)) {
      target.setBaseUrl(null);
      return;
    }
```
Maybe it's enough to add _target.setBaseUrl(null);_ before return?

Reply:
If we remove the return statement, execution will continue to the subsequent code, potentially resulting in an Exception: "URI is not absolute" when attempting to create a URL from an empty string.

Reply:
It was not about removing of return statement. Okay, never mind